src/test/java/com/epam/test_generator/entity/SuitTest.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.test_generator.entity;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import com.epam.test_generator.entities.Case;
+import com.epam.test_generator.entities.Status;
+import com.epam.test_generator.entities.Step;
+import com.epam.test_generator.entities.StepType;
+import com.epam.test_generator.entities.Suit;
+import com.epam.test_generator.entities.Tag;
+import com.epam.test_generator.entities.api.SuitTrait;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SuitTest {
+
+    private static final long SIMPLE_SUIT_ID = 1L;
+    private static final long SIMPLE_CASE_ID = 2L;
+    private static final long SIMPLE_STEP_ID = 3L;
+    private SuitTrait suit;
+
+    private Case caze;
+
+    @Before
+    public void setUp() {
+        Step step = new Step(SIMPLE_STEP_ID, 1, "Step desc", StepType.GIVEN, "Comment",
+            Status.NOT_RUN);
+
+        List<Step> listSteps = new ArrayList<>();
```

Comment:
better name variable *steps* or *stepList*, current name is more like command (verb)